docs/4.3/admin-guide.md

```diff
@@ -1588,6 +1588,13 @@ achieve highly available deployments. You must take steps to protect access to
 `etcd` in this configuration because that is where Teleport secrets like keys
 and user records will be stored.
 
+!!! warning "IMPORTANT"
+
+    `etcd` can only currently be used to store Teleport's internal database in a highly-available
```

Comment:
I guess we don't really have a story for this yet. We chatted about this with a customer this week about possibly adding Elastic Search as an option for audit events then we'll have a better story for on-prem HA customers.

Reply:
Yeah, we should definitely come up with some kind of supported non-proprietary method for event storage.